Rephrase wire format parsing error description (#260)
MikeBishop/dns-alt-svc: draft-ietf-dnsop-svcb-https.md

Clients MUST consider an RR malformed if:

> * the parser reaches the end of the RDATA in a non-terminal state.

Author: ```suggestion
* the end of the RDATA occurs within a SvcParam.
```
